src/encoded/types/file.py

```diff
@@ -282,6 +296,8 @@ def analysis_step_version(self, request, root, step_run=None):
 
     @calculated_property(schema={
         "title": "Output category",
+        "description": "The overall catagory of the file content.",
```

Comment:
s/catagory/category/.   Maybe should read "Broad classification of the file content".  Change title too?